app/components/ShowsLister.vue

```diff
@@ -15,7 +15,7 @@
           </div>
           <div class="show-desc p-6">
             <div class="showdescription">
-              {{ truncate(show.description, 200) }}
+              {{ show.description && truncate(stripHTMLTags(show.description), 200) }}
```

Comment:
What's the logical operator `&&` doing here? Does it check if `show.description` is empty? Can we use a more explicit notation instead, e.g., using `v-if` or the ternary operator?

Reply:
Yes, otherwise it would break if it tries to run the functions if show.description is not yet initialized
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND

Reply:
Interesting semantics. Thanks for the explanation.